test/unit/clitests_helper.js

```diff
@@ -18,6 +18,9 @@ import { isNodeJS } from "../../src/shared/is_node.js";
 import { PDFNodeStream } from "../../src/display/node_stream.js";
 import { setPDFNetworkStreamFactory } from "../../src/display/api.js";
 
+// Sets longer timeout, similar to `jasmine-boot.js`.
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
+
```

Comment:
As far as I can tell this timeout has been too low since "forever", but apparently it hasn't really been an issue previously.